docs/javascript/browser-apis-and-jquery.md

```diff
@@ -224,6 +224,125 @@ The `document.getElementById()` method allows you to select an HTML element by i
 <figcaption></figcaption>
 </figure>
 
+### Getting an element by `class name`
+
+The `document.getElementsByClassName()` method allows you to return the selected HTML elements by it's class attribute:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div>
+      <div class="square"></div>
+      <div class="circle"></div>
+      <div class="square"></div>
+      <div class="square"></div>
+      <div class="circle"></div>
+    </div>
+    <script>
+      // Return all elements with the class name of `square`
+      var getSquares = document.getElementsByClassName("square");
+
+      // Add an image to all div's with class name `square`
+      getSquares.style.backgroundImage = "<img src='square.png'>";
+    </script>
+  </body>
+</html>
+```
+
+### Getting an element by `tag name`
+
+The `document.getElementsByTagName()` method allows you to return the selected HTML elements by it's tag name:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div>
+      <div class="square"></div>
+      <div class="circle"></div>
+      <div class="square"></div>
+      <div class="square"></div>
+      <div class="circle"></div>
+    </div>
+    <script>
+      // Return all div elements
+      var getAllElements = document.getElementsByTagName("div");
+
+      // Add border color to all div tags
+      getAllElements.style.borderColor = "yellow";
```

Comment:
Same issue as `getElementsByClassName()`. Also, variable names shouldn't start with a verb, as that makes them look like a function. `divElements` would be a better name.